packages/components/popover/package.json

```diff
@@ -22,6 +22,6 @@
     "directory": "packages/components/popover"
   },
   "dependencies": {
-    "@radix-ui/react-popover": "1.0.6-rc.13"
+    "@radix-ui/react-popover": "1.0.6"
```

Comment:
:+1:

Reply:
The final release was done some time after the Popover was merged to master.